src/Library/demos/Avatar/main.py

```diff
@@ -0,0 +1,31 @@
+import gi
```

Comment:
For some reasons the Python demo does not load the default avatar image for me, showing just a placeholder. But I can see in the inspector that the correct icon is placed there, so either something is wrong in my dev setup or this is an issue with the previewer process or PyGObject.